packages/client/test/end-to-end/publish-subscribe.test.ts

```diff
@@ -67,19 +71,8 @@ describe('publish-subscribe', () => {
     })
 
     beforeEach(async () => {
-        publisherClient = new StreamrClient({
-            ...CONFIG_TEST,
-            auth: {
-                privateKey: publisherPk
-            }
-        })
-        subscriberClient = new StreamrClient({
-            ...CONFIG_TEST,
-            auth: {
-                privateKey: subscriberWallet.privateKey
-            }
-        })
-
+        publisherClient = createTestClient(publisherPk, 'publisher', 15656)
```

Comment:
Would it be possible that we'd allocate a random free port for the test client, so that we don't need to specify these magic numbers in the tests.

Reply:
This might prove be more difficult than expected as the DHT package may take some time to start the ws server. Ie. a test util sees that a port is free but has already allocated the port for another NetworkNode. The util would probably need to keep state between tests for which ports it has allocated.

Reply:
Does our websocket server allocate a random port if we use `0` as the port number? Or do not provide any port to the server? (https://stackoverflow.com/questions/28050171/nodejs-random-free-tcp-ports)